README.md

```diff
@@ -974,13 +977,12 @@ If you wish to use an environment variables, use set:
 
 ```
 set VERBOSE=1
-cmake --build . --config Release --target tests
+cmake --build . --config Release --target test
```

Comment:
`ctest`?

Reply:
We should never build --target test.  It doesn't work on MSVC builds.

Reply:
Why not, [works fine on CI for MSVC](https://github.com/Exiv2/exiv2/blob/29cc98118003a9f1c632bde9bd706f9483de4e8b/.github/workflows/on_PR_windows_matrix.yml#L90), no?

Reply:
Doesn't work with Visual Studio.  It says "test.vcxproj" doesn't exist.  That's right, it doesn't.   Visual Studio builds the executables.  It doesn't do the test.

I think the CI must be using ctest and it will need `ctest -C Release` for MSVC.